sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/api/CaseSteps.java

```diff
@@ -17,17 +17,116 @@
+  public CaseSteps(
+      CaseHelper caseHelper,
+      PersonsHelper personHelper,
+      ApiState apiState,
+      CaseApiService caseApiService,
+      CommunityHelper communityHelper,
+      CountryHelper countryHelper,
+      BodyResources bodyResources,
+      PostCaseBodyService bodyService,
+      PostPersonBodyService personBodyService) {
+
+    Given(
+        "I create a person",
+        () -> {
+          bodyResources.setBody("[" + personBodyService.generatePostPersonBody() + "]");
+          personHelper.pushPerson("push", bodyResources.getBody());
+          System.out.println("Create person response = " + apiState.getResponse().getStatusCode());
+        });
+
+    Given(
+        "I try to enter invalid user for a new case",
+        () -> {
+          bodyResources.setBody("[" + bodyService.generatePostCaseBodyTooOld() + "]");
+        });
+
+    Given(
+        "I try to enter valid data for a new case",
+        () -> {
+          bodyResources.setBody("[" + bodyService.generatePostCaseBodyValid() + "]");
+        });
+
+    Given(
+        "I try to enter invalid disease for a new case",
+        () -> {
+          bodyResources.setBody("[" + bodyService.generatePostCaseBodyUnknownDisease() + "]");
+        });
+
+    Given(
+        "I try to enter all lower case healthFacility for a new case",
+        () -> {
+          bodyResources.setBody("[" + bodyService.generatePostCaseBodyWithInvalidFacility() + "]");
+        });
+
+    When(
+        "I create a new case",
+        () -> {
+          caseHelper.postCases("push", bodyResources.getBody());
+        });
+
+    Then(
+        "I get the error message TOO_OLD",
+        () -> {
+          assertThat(apiState.getResponse().getStatusCode()).toString().startsWith("2");
+          assertThat(apiState.getResponse().getBody().asString())
+              .ignoringCase()
+              .equals(String.valueOf(ErrorMessages.TOO_OLD));
+        });
+
+    Then(
+        "I get a general error message ERROR",
+        () -> {
+          assertThat(apiState.getResponse().getStatusCode()).toString().startsWith("2");
+          assertThat(apiState.getResponse().getBody().asString())
+              .ignoringCase()
+              .equals(String.valueOf(ErrorMessages.ERROR));
+        });
+
+    Then(
+        "I get successful response back",
+        () -> {
+          assertThat(apiState.getResponse().getStatusCode()).toString().startsWith("2");
+          assertThat(apiState.getResponse().getBody().asString()).contains(String.valueOf("OK"));
+        });
+
+    Then(
+        "I get the error message Unknown disease and all the valid values are shown",
+        () -> {
+          assertThat(apiState.getResponse().getStatusCode()).toString().equalsIgnoreCase("400");
+          assertThat(apiState.getResponse().getBody().asString())
+              .contains(String.valueOf(ErrorMessages.UNKNOWN_DISEASE));
+        });
+
+    Then(
+        "I can query case by UUID",
```

Comment:
there is an existent approach in framework to get a case by UUID

Reply:
I did not find it, if you can pointed out to me, no need to duplicate. If I can use it, I will.

Reply:
So, in framework we already have API created: contacts, persons, cases etc. In ApiState.class you will see stored all recently created objects via API calls. 
So by using this: apiState.getCreatedCase().getUuid(), we can get the uuid of most of the stored objects/pojos.

Also, this method is redundant.. I don't see why are we adding a step in a test to print the UUID if we're not using it in any other step. If we need it for debug, we can store it in log files, or discuss an approach to add log.debug("message") to help us in future.